0_web_functions.R

```diff
@@ -187,6 +204,11 @@ read_web_input_file <- function(input_file_path){
   
   if(data_check(input_file) == FALSE){
     warning("Input file not readable")
+    ifelse(nrow(input_file) == 0,
+           write_log(msg = "Input file has 0 rows. Please ensure the uploaded file is not empty."),
+           write_log(msg = "Input file could not be transformed into a data.frame.
```

Comment:
These are two different error messages - is the second meant to be the "stop"?

Reply:
ehm, no. interesting case. Will have a look

Reply:
Ah, I remember. the messages are two diff arguments to the ifelse() function.
If data check is FALSE AND the input file has zero rows, you get the first message, If data check is FALSE AND the input file does not have exactly nrows == 0, the second message highlights that the file could not even be transformed into a data frame. within data_check() that reflects is.data.frame == F
So this is expected